Gemfile

```diff
@@ -1,14 +1,11 @@
 source 'https://rubygems.org'
 
-# The Ruby version should match the lowest acceptable version of the application
-raise "Ruby 3.1 required" unless RUBY_VERSION >= "3.1" && RUBY_VERSION < "3.2"
```

Comment:
I dropped this because it breaks renovate - it was more about making devs run 3.1 so I removed it for now.

Reply:
Does `ruby '~> 3.1.0'` do what you want?

Reply:
> Does `ruby '~> 3.1.0'` do what you want?

Wait you can do the twiddle wack with the ruby command?  I'll give it a try.

Reply:
yes, https://bundler.io/guides/gemfile_ruby.html

I think we did it this way in manageiq because we wanted a way to warn on untested rubies we wanted to allow but raise on others we explicitly do not want to run on.

Reply:
No, I just added this line this past week. I tried the ruby command first, but I never thought to use ~>, so it was forcing me to use an exact z stream, and I didn't like that.